src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/forgotpassword/ForgotPasswordPage.java

```diff
@@ -0,0 +1,10 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormPage;
+
+public interface ForgotPasswordPage extends FormPage {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1214)